src/components/calcite-dropdown/calcite-dropdown.e2e.ts

```diff
@@ -753,6 +753,60 @@ describe("calcite-dropdown", () => {
     expect(await dropdownWrapper.isVisible()).toBe(false);
   });
 
+  describe("opens the dropdown with click, enter, or space", () => {
+    it("opens when dropdown-trigger is a button", async () => {
+      const page = await newE2EPage();
```

Comment:
nitpick: You can use `newE2EPage({html: "<html>"})` and then you won't have to call `setContent`.